<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" rowspan="4">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Partícula</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="11" column="1" colspan="2">
            <widget class="QSpinBox" name="blue_color_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="rojo">
             <property name="text">
              <string>Red:</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0" colspan="3">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QSpinBox" name="red_color_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="2">
            <widget class="QSpinBox" name="origen_x_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QSpinBox" name="velocidad_spinBox">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QSpinBox" name="origen_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="id">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="3">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar Inicio</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="verde">
             <property name="text">
              <string>Green:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="azul">
             <property name="text">
              <string>Blue:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="destinoY">
             <property name="text">
              <string>Destino en y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="origenX">
             <property name="text">
              <string>Origen en x:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QSpinBox" name="id_spinBox">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="6" column="1" colspan="2">
            <widget class="QSpinBox" name="destino_y_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="3">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Color (RGB):</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="velocidad">
             <property name="text">
              <string>Velocidad</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="destinoX">
             <property name="text">
              <string>Destino en x:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="origenY">
             <property name="text">
              <string>Origen en y:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1" colspan="2">
            <widget class="QSpinBox" name="green_color_spinBox">
             <property name="maximum">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QSpinBox" name="destino_x_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Información Registrada</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="salida_plainTextEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Salida</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="algoritmos_plainTextEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_tabla_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="ordenar_distan_pushButton">
          <property name="text">
           <string>Ordenar por Distancia</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla_tableWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Ingrese el Id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="ordenar_id_pushButton">
          <property name="text">
           <string>Ordenar por Id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="ordenar_veloci_pushButton">
          <property name="text">
           <string>Ordenar por velocidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionGrafo"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionRecorrido_Profundidad_Amplitud"/>
    <addaction name="actionPrim"/>
    <addaction name="actionKruskal"/>
    <addaction name="actionDijkstra"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionGrafo">
   <property name="text">
    <string>Grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionRecorrido_Profundidad_Amplitud">
   <property name="text">
    <string>Busqueda (Profundidad/Amplitud)</string>
   </property>
  </action>
  <action name="actionPrim">
   <property name="text">
    <string>Prim</string>
   </property>
  </action>
  <action name="actionKruskal">
   <property name="text">
    <string>Kruskal</string>
   </property>
  </action>
  <action name="actionDijkstra">
   <property name="text">
    <string>Dijkstra</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
